ui/components/app/transaction-detail-item/index.scss

```diff
@@ -7,6 +7,15 @@
     display: flex;
     justify-content: space-between;
     grid-gap: 5px;
+
+    &__subText {
```

Comment:
What are these changes in response to? Or were they just missed in a previous PR?

Reply:
Yep they were missed out in previous one